examples/broadcasting_your_own_methods.py

```diff
@@ -47,25 +47,29 @@
 positions.loc[:, "y", :, "AEON3B_TP1"] -= 100.0
 positions.loc[:, "y", :, "AEON3B_TP2"] += 100.0
 
+# %%
+
 fig, ax = plt.subplots(1, 1)
 for mouse_name, col in zip(
     positions.individuals.values, ["r", "g", "b"], strict=False
 ):
-    ax.plot(
-        positions.sel(individuals=mouse_name, space="x"),
-        positions.sel(individuals=mouse_name, space="y"),
+    plot_trajectory(
+        positions,
+        individual=mouse_name,
+        keypoints="centroid",
+        ax=ax,
         linestyle="-",
         marker=".",
-        markersize=2,
+        s=2,
         linewidth=0.5,
         c=col,
         label=mouse_name,
     )
-    ax.invert_yaxis()
-    ax.set_xlabel("x (pixels)")
-    ax.set_ylabel("y (pixels)")
-    ax.axis("equal")
-    ax.legend()
+ax.invert_yaxis()
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.axis("equal")
```

Comment:
These 3 lines are not needed because `plot_trajectory` sets these values by default.
```suggestion
```